packages/material-ui-lab/src/Autocomplete/Autocomplete.test.js

```diff
@@ -1610,5 +1610,46 @@ describe('<Autocomplete />', () => {
       expect(handleChange.args[1][1]).to.equal(options[0]);
       expect(handleChange.args[1][2]).to.equal('mouse');
     });
+
+    it('should mantain list box open clicking on input when it is not empty', () => {
+      const handleChange = spy();
+      const { queryByRole, getByRole, getAllByRole } = render(
+        <Autocomplete
+          onHighlightChange={handleChange}
+          options={['one']}
+          renderInput={(params) => <TextField {...params} />}
+        />,
+      );
+      const textbox = getByRole('textbox');
+      expect(queryByRole('presentation')).to.equal(null);
+      fireEvent.mouseDown(textbox); // Open listbox
+      expect(queryByRole('presentation')).to.not.equal(null);
+      const options = getAllByRole('option');
+      fireEvent.click(options[0]);
+      expect(queryByRole('presentation')).to.equal(null);
+      fireEvent.mouseDown(textbox); // Open listbox
+      expect(queryByRole('presentation')).to.not.equal(null);
+      fireEvent.mouseDown(textbox); // Remain open listbox
+      expect(queryByRole('presentation')).to.not.equal(null);
+    });
+
+    it('should toggle list box when input is empty', () => {
+      const handleChange = spy();
+      const { queryByRole, getByRole } = render(
+        <Autocomplete
+          onHighlightChange={handleChange}
+          options={['one']}
+          renderInput={(params) => <TextField {...params} />}
+        />,
+      );
+      const textbox = getByRole('textbox');
+      expect(queryByRole('presentation')).to.equal(null);
```

Comment:
Using `queryByRole('presentation')` sounds wrong. 
```suggestion
      expect(combobox).to.have.attribute('aria-expanded', 'false');
```

Reply:
I can't use geBy* because if the node doesn't exist it will throw exception and in that case I wanted to test it